docs/components/Rating/rating.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav>
+        <div class="nav-contents">
+          <a href="/index.html">
+            <img
+              class="home-svg"
+              src="https://api.iconify.design/twemoji:letter-m.svg"
+              alt="home"
+            />
+          </a>
+          <a class="link" href="/index.html">Installation</a>
+          <a class="link" href="/docs/components/alert/alert.html">Documentation</a>
+          <a class="link" href="https://github.com/Muzammil-khan710/Component-Library">GitHub</a>
+        </div>
+      </nav>
+
+    <main class="container">
+        <div class="left-container">
+          <h2>Components</h2>
+          <a class="link" href="../alert/alert.html">Alert</a>
+          <a class="link" href="../button/button.html">Button</a>
+          <a class="link" href="../avatar/avatar.html">Avatar</a>
+          <a class="link" href="../badge/badge.html">Badges</a>
+          <a class="link" href="../cards/cards.html">Cards</a>
+          <a class="link" href="../image/image.html">Images</a>
+          <a class="link" href="../input/input.html">Input</a>
+          <a class="link" href="../snackbar/snackbar.html">Snackbar</a>
+          <a class="link" href="../list/list.html">List</a>
+          <a class="link" href="../Rating/rating.html">Rating</a>
+        </div>
+
+        <div class="snackbar-container">
+            <h2 class="t-cntr">Rating</h2>
+            <p>A rating is an evaluation or assessment of something, in terms of quality, quantity, or some combination of both .</p>
+
+            <div class="star-container">
+                <ul class="ul-ls">
+                    <li class="star orange">&#x2605;</i></li>
+                    <li class="star orange">&#x2605;</i></li>
+                    <li class="star">&#x2605;</i></li>
+                    <li class="star">&#x2605;</i></li>
+                    <li class="star">&#x2605;</i></li>
```

Comment:
Question : 
1. Why are we giving color to star like this here? nth child could be used here in combination with star-container class.
2. Why is there a closing tag for i tag?

Reply:
1. As i am not using JS so i have give color to the classes.. I will update when i will be adding JS to my Component Library.
2. searched online for star icon and found it . Didn't saw that i tag. Thanks for highlighting